python/tank/deploy/tank_commands/core_upgrade.py

```diff
@@ -151,38 +154,38 @@ def _run(self, log, suppress_prompts, core_version):
         log.info("https://support.shotgunsoftware.com/entries/96141707")
         log.info("https://support.shotgunsoftware.com/entries/96142347")
         log.info("")
-        log.info("")    
-        
-        installer = TankCoreUpdater(code_install_root, log, core_version)
+        log.info("")
+
+        installer = TankCoreUpdater(code_install_root, log, core_constraint_pattern)
         cv = installer.get_current_version_number()
-        lv = installer.get_update_version_number()
+        nv = installer.get_update_version_number()
```

Comment:
lv stood for `latest version` originally. it actually stands for new `version now` that we can upgrade to any version so I changed this accordingly.